public/app/views/shared/_digital.html.erb

```diff
@@ -1,8 +1,4 @@
-<% if false %><!-- comment -->
-  expects 'rep_fv' as a file_version hash when AppConfig[:enable_representative_file_version]
-  otherwise expects to-be-deprecated 'dig_objs' as an array of hashes
```

Comment:
The `dig_objs.present?` (display all file versions stuff) is still around in this template but I'm going to assume that's ok ...

Reply:
I think so - we need to revaluate the templates and continue to synthesize the old and new logic.